javascript/elements/updates_for_element.js

```diff
@@ -61,15 +61,20 @@ export default class UpdatesForElement extends SubscribingElement {
       blocks[i].setAttribute('updating', 'updating')
 
       if (!html.hasOwnProperty(url(blocks[i]))) {
-        const response = await fetch(url(blocks[i]), {
-          headers: {
-            'X-Cable-Ready': 'update'
-          }
-        })
-          .then(handleErrors)
-          .catch(e => console.error(`Could not fetch ${url(blocks[i])}`))
-        if (response === undefined) return
-        html[url(blocks[i])] = await response.text()
+        try {
+          const response = await fetch(url(blocks[i]), {
+            headers: {
+              'X-Cable-Ready': 'update'
+            }
+          })
+          if (response === undefined) return
```

Comment:
`response == undefined` or `response == null` is probably preferred here! 

```js
null === undefined // False!
null == undefined // True!
``` 

Not a huge deal, but one of those fun quirks to be in mind. Granted, this will raise an error once it gets to the next line for `handleErrors`.

Reply:
Ah, nice one! I'm thinking about how to best extract that whole functionality to a custom `fetch` routine